### PR TITLE
B-08224:

### PR DESCRIPTION
**Pull request by markyd-XXI**
- Creating stories list view to display parent stories for the logged in user.
 - Creating story card component to show story details.
 - Updated story services for all story capabilities.

### CODE REVIEW

File: src/services/StoryServices.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    return apiClient.get("stories");
+  },
+  getStoriesByUserId(userId) {
+    return apiClient.get(`stories/${userId}`);
+  },
+  getStory(id) {
+    return apiClient.get(`stories/${id}`);
+  },
+  addStory(userId) {
+    return apiClient.post(`stories/${userId}`);
```
Comment (n-grounds) [inline src/services/StoryServices.js:14]: where does the body of the POST come from?

File: src/services/StoryServices.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  },
+  updateStory(storyId, story) {
+    return apiClient.put(`stories/${storyId}`, story);
+  },
+  deleteStory(storyId) {
+    return apiClient.delete(`stories/${storyId}`);
+  },
+  extendStory(storyId) {
+    return apiClient.post(`stories/extend/${storyId}`);
```
Comment (n-grounds) [inline src/services/StoryServices.js:23]: where does the body of the POST come from?

File: src/views/StoriesList.vue [hunk 3]
```diff
@@ -89,7 +89,7 @@ function closeSnackBar() {
       <v-row align="center" class="mb-4">
         <v-col cols="10"
           ><v-card-title class="pl-0 text-h4 font-weight-bold"
-            >Recipes
+            >Stories
```
Comment (n-grounds) [inline src/views/StoriesList.vue:92]: why is the greater than signs on the line after the matching less than sign?

File: src/components/StoryCardComponent.vue [hunk 4]
```diff
@@ -0,0 +1,46 @@
+<script setup>
+import { ref } from "vue";
+
+const showStory = ref(false);
+
+const props = defineProps({
```
Comment (n-grounds) [inline src/components/StoryCardComponent.vue:6]: maybe I'm just missing it, but is this props const ever used anywhere?

File: src/services/StoryServices.js (hunk 1 above)
Comment (markyd-XXI) [reply src/services/StoryServices.js:14]: This is mostly just a stub and it's not used anywhere. I believe Maurice will have an implementation incoming with a body.

File: src/services/StoryServices.js (hunk 2 above)
Comment (markyd-XXI) [reply src/services/StoryServices.js:23]: Same here, this is mostly just a stub and it's not used anywhere. I believe Maurice will have an implementation incoming with a body.

File: src/components/StoryCardComponent.vue (hunk 4 above)
Comment (markyd-XXI) [reply src/components/StoryCardComponent.vue]: I removed it. Thanks for the catch.

File: src/views/StoriesList.vue (hunk 3 above)
Comment (markyd-XXI) [reply src/views/StoriesList.vue]: I guess I didn't have the Vue formatter configured in vscode. Thanks,

**Pull request merged**